src/description.jl

```diff
@@ -17,6 +17,10 @@ function description(interval::AnchoredInterval{P, ZonedDateTime}, s::String) wh
     )
 end
 
+function description(dt::TimeType, p::Period, suffix::String)
+    return string(prefix(p), suffix, " ", string(dt))
```

Comment:
You actually don't need the interior call to `string` here; each argument to `string` will be converted as though it itself was wrapped in `string`.

Reply:
whoops, missed that. Thanks!